### PR TITLE
[Doc] when \n in in variable, then setValue will not work well, 

### LINKED ISSUE #15136: [Bug] [setValue] if "\n" in variable, then setValue function will work bad
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

In python task , if a variable has "\n"， then use setValue to make a Parameter for next node,  the variable will not work well 

node_one 
```python
s = "hello \n world"
print("${setValue(message=%s)}"%s)
```

node_two
```shell
echo "${message}"

echo "hi"
```

check runtime log 

node_one 
```
[INFO] 2023-11-08 10:51:02.588 +0800 - process start, process id is: 10376
[INFO] 2023-11-08 10:51:03.588 +0800 -  -> 
	 world)}
[INFO] 2023-11-08 10:51:03.590 +0800 - process has exited. execute 
```

node_two
```
[INFO] 2023-11-08 10:51:04.374 +0800 - process start, process id is: 30190
[INFO] 2023-11-08 10:51:05.374 +0800 -  -> 
	
	hi
[INFO] 2023-11-08 10:51:05.375 +0800 - process has exited. execute
```

### What you expected to happen

In node_two, when `echo "${message}"`  will show the  Parameter set in pre task,  in this case, want show “hello \n world"

### How to reproduce

create a workflow, the first task use python task, make a variable include "\n", then use setValue to make a  Parameter

### Anything else

_No response_

### Version

3.2.x

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kevinkelin**
when '\n' indicates the presence of a variable in python task,then needs to use 'repr(value)'

## Purpose of the pull request
close #15136 

Use python task need notice

## Brief change log
In parameter chapter, use setValue func for set a parameter in all flow, when the value contains `\n` , we must use `repr(value)` for format string in python task.

## Verify this pull request
This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by qingwli — APPROVED** [review]
LGTM, Wait for CI

**Comment by qingwli** [comment]
Please Run 'mvn spotless:apply' to fix these violations.

**Review by SbloodyS — APPROVED** [review]
+1

**Comment by SbloodyS** [comment]
Please run `mvn spotless:apply` to fix CI. @kevinkelin

File: docs/docs/en/guide/parameter/context.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ For example
 
 ![python_context_param](../../../../img/new_ui/dev/parameter/python_context_param.png)
 
+
+Attention: When the variable value contains the `\n` identifier, such as ` value = "hello \n world" `, value needs to be carried out in a special way. You need to use `print('${setValue(key=%s)}' % repr(value))`, otherwise the argument cannot be passed to the subsequent flow.
```
Comment (Radeity) [inline docs/docs/en/guide/parameter/context.md:96]: Some humble suggestion, what do you think using "... value should remain unescaped, such as using `print('${setValue(key=%s)}' % repr(value))` ..." which is more clear.

**Pull request merged**